Documentation/teaching/info/vm.rst

```diff
@@ -11,15 +11,20 @@ virtual machine.
 In order to run and use the virtual machine the following packages are required
 on a Debian/Ubuntu system:
 
+* ``flex``
+* ``bison``
```

Comment:
Please add these packages to the Dockerfile as well.

Reply:
@Sergiu121 
What dockerfile are you referring to?
In the one for documentation I don't think it's necessary, and in the one for the kernel image I see that it already exists these packages.

Reply:
@Sergiu121 , it's already in the [Dockerfile](https://github.com/linux-kernel-labs/linux/blob/master/tools/labs/docker/kernel/Dockerfile#L24), as Gabi said.

Reply:
My bad. You're right